src/controllers/playback/videoosd.js

```diff
@@ -1224,12 +1224,14 @@ define(["playbackManager", "dom", "inputManager", "datetime", "itemHelper", "med
             return null;
         }
 
+        let playPauseClickTimeout;
```

Comment:
are we aware that this breaks older browsers like WebOS? :) they don't handle `let`